test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertFrom-StringData.Tests.ps1

```diff
@@ -69,9 +69,9 @@ bazz = 2
 Describe "Delimiter parameter tests" -Tags "CI" {
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-windows`
TestAppDomainProcessExitEvenHandlerNotLeaking
```powershell
Assert.DoesNotContain() Failure
Found:    (filter expression)
In value: Delegate[] [EventHandler { Method = Void DisposeOnShutdown(System.Object, System.EventArgs), Target = null }, EventHandler { Method = Void CurrentDomain_ProcessExit(System.Object, System.EventArgs), Target = null }]
   at PSTests.Sequential.RunspaceTests.TestAppDomainProcessExitEvenHandlerNotLeaking() in D:\a\1\s\test\xUnit\csharp\test_Runspace.cs:line 127
```